src/utils/dispute-utils.js

```diff
@@ -140,6 +146,12 @@ export function getDisputeTimeLine(dispute, courtConfig) {
   return timeLine
 }
 
+/**
+ * @param {Object} dispute The dispute querying current phase and transition of
+ * @param {Object} courtConfig The court configuration
+ * @param {Date} nowDate Current date
```

Comment:
`nowDate` seems like it could be renamed to something else; otherwise we could just use `dayjs()` in the function itself?

Reply:
Yeah, changed the naming as well as the type in favor of using `dayjs()`. I think it makes more sense to stick with passing the date as parameter since we are passing the result of `useNow` in some places e.g in dispute hooks.

Also changed `useNow` to set a `dayjs` object as opposed to the `Date` object so we can use it transparently.